rvps/README.md

```diff
@@ -172,13 +172,14 @@ Then it will say something like
 [2023-03-09T04:44:11Z INFO  rvps_client] Register provenance succeeded.
 ```
 
-Let's then query the reference value
+Let's then query the reference values
 ```bash
-rvps-tool query --name test-binary-1 --addr http://$RVPS_ADDR
+rvps-tool query --addr http://$RVPS_ADDR
```

Comment:
Yeah I think I removed this when we changed the interface of the RVPS. Thanks for fixing this.